tests/test_commit.py

```diff
@@ -62,6 +62,21 @@ def edit_table(dbcur):
     return pk_del
 
 
+def _count_tracking_table_changes(db, working_copy, layer):
+    with db:
+        cur = db.cursor()
+        cur.execute(
+            f"SELECT COUNT(*) FROM {working_copy.TRACKING_TABLE} WHERE table_name=?;",
+            [layer],
+        )
+        change_count = cur.fetchone()[0]
+        del cur
+    return change_count
```

Comment:
```suggestion
        return cur.fetchone()[0]
```



don't need the `del` because it gets garbage-collected on return anyway

Reply:
Done